lib/cocoapods/target.rb

```diff
@@ -298,6 +298,23 @@ def info_plist_entries
       {}
     end
 
+    # @return [String] Bundle Identifier found in the custom Info.plist entries
+    #
+    def info_plist_bundle_id
```

Comment:
can this be move to `pod_target_installer`? I think the logic here (including the warning) makes more sense to be localized to the installer and not exposed on every instance of `PodTarget`

Reply:
@amorde, thanks for the input, moved the method to `pod_target_installer`.

Reply:
looks like it's now in `pod_target`, not `pod_target_installer`

Reply:
Apologies for that!  I've made the expected update now.